crate/test_server/benches/benches.rs

```diff
@@ -239,7 +241,10 @@ fn bench_rsa_encrypt(
             .get_or_try_init(start_default_test_kms_server)
             .await
             .unwrap();
-        let kms_rest_client = ctx.owner_client_conf.initialize_kms_client().unwrap();
+        let kms_rest_client = ctx
+            .owner_client_conf
+            .initialize_kms_client(None, None)
+            .unwrap();
```

Comment:
panic risk.

Reply:
thanks for notifying it. As discussed, unwrap() are allowed only in tests and benches.